### PR TITLE
fix(select): disable state color for single option

### PR DESCRIPTION
**Pull request by valentin-mladenov**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?
1. Select disable color is using `cds-alias-typography-disabled`
2. A single disabled select option doesn't have proper colors set

Issue Number: CDE-2138 and CDE-2145

## What is the new behavior?
1. Select disable color is using `cds-alias-object-interaction-color-disabled`
2. A single disabled select option have proper colors set

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by dtsanevmw — CHANGES_REQUESTED** [review]
This mixin is using `cds-alias-typography-disabled` as underlying token but for this case as it's interaction item like in the dropdown it should use `cds-alias-object-interaction-color-disabled`

**Comment by valentin-mladenov** [comment]
> This mixin is using `cds-alias-typography-disabled` as it's meaning is to be for the labels but for this case as it's interaction item like in the dropdown it should use `cds-alias-object-interaction-color-disabled`

@dtsanevmw Changing the rules to use token instead of the mixin is more broader task. The change will affect all select/option disable variations.

This PR only targets single option disable state to be aligned with the rest variants.

File: .storybook/stories/select/select.stories.ts [hunk 1]
```diff
@@ -37,7 +37,9 @@ const selectTemplate: StoryFn = args => ({
     <clr-select-container>
       <label>Options</label>
       <select clrSelect>
-        <option *ngFor="let _ of createArray(optionCount); let i = index" [value]="i + 1">Option {{ i + 1 }}</option>
+        <option *ngFor="let _ of createArray(optionCount); let i = index" [value]="i + 1" [disabled]="i === 1">
```
Comment (kevinbuhmann) [inline .storybook/stories/select/select.stories.ts:40]:
```suggestion
        <option *ngFor="let _ of createArray(optionCount); let i = index" [value]="i + 1" [disabled]="i === 0">
```

Should this one be consistent with the multiple select story?

Comment (valentin-mladenov) [reply .storybook/stories/select/select.stories.ts]: Doesn't really matter.

I changed the single selection story. 
Commit e8fb100

**Review by dtsanevmw — APPROVED** [review]

**Pull request merged**